packages/react-components/src/localization/locales/de-DE/strings.json

```diff
@@ -106,4 +106,4 @@
     "screenIsBeingSharedMessage": "Sie teilen Ihren Bildschirm",
     "screenShareLoadingMessage": "Der Bildschirm von {participant} wird geladen"
   }
-}
\ No newline at end of file
+}

```

Comment:
Spurious diffs... I've had these too. The touchdown build removes trailing newlines from these files :(